.ci/scripts/install_windows.bat

```diff
@@ -1,6 +1,16 @@
-REM SPDX-FileCopyrightText: 2020 Intel Corporation
-REM
-REM SPDX-License-Identifier: MIT
+rem Copyright 2020 Intel Corporation
```

Comment:
Looks as though this file is unused in all the various CIs, @Alexsandruss should we make a PR to delete this, or have @homksei do it here?

Reply:
It's better to remove this file here, @homksei.

Reply:
Done